src/machines/java/com/enderio/machines/common/recipe/AlloySmeltingRecipe.java

```diff
@@ -119,11 +119,11 @@ public RecipeType<?> getType() {
      * The recipe container.
      * This acts as additional context.
      */
-    public static class Container extends RecipeWrapper {
+    public static class ContainerWrapper extends RecipeWrapper {
```

Comment:
What was the reason for renaming this class?

Reply:
Duplicate Name. This would clash with the implements Container for the Sub Wrapper. Wait mvmd, I've moved that one, no it's not needed but to distinguish between the Container interface and the subwrapper

Reply:
Hmm, unsure if thats a good idea or not because we've called it "Container" for all of the other recipes. Maybe we could rename all the recipe containers "RecipeContainer" instead? Might be a good way to differ it from the interface

Reply:
I don't think it's necessary, it's just needed for the distinction between this specific wrapper and the subwrapper, I wouldn't change them all, as this only adds to the readability in this case

Reply:
Makes sense, thanks a lot!